src/app/components/ExerciseCard/ExerciseCard.stories.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import ExerciseCard from './ExerciseCard';
+
+export default {
+  title: 'Components/ExerciseCard',
+  component: ExerciseCard,
+};
+
+export const Card = (): JSX.Element => (
+  <ExerciseCard thumbnail="pushups.jpeg" name="Push Ups" category="muscle" />
+);
```

Comment:
More stories would be awesome to see how your component behaves with different properties.